app/(drawer)/(tabs)/library/music/playlists/[playlistId]/index.tsx

```diff
@@ -0,0 +1,172 @@
+
+  const handleRowPress = async (index: number) => {
+    await loadTrackList({
+      trackList: playlistTracks,
+      trackListId: playlistId as string,
+      startIndex: index,
+      playerTitle: playlistTitle as string,
+    });
+  };
+
+  const handlePlayPausePress = () => {
+    if (isThisTrackListLoaded) {
+      return togglePlayPause();
+    }
+
+    return loadTrackList({
+      trackList: playlistTracks,
+      trackListId: playlistId as string,
+      startIndex: 0,
+      playerTitle: playlistTitle as string,
+    });
+  };
+
+  const handleMorePress = () => {
+    setMoreIsOpen(true);
+  };
+
+  return playlistTracks ? (
+    <View style={{ height: "100%", paddingTop: 8, gap: 8 }}>
+      <View
+        style={{
+          flexDirection: "row",
+          alignItems: "center",
+          gap: 24,
+          padding: 8,
+        }}
+      >
+        {!!playlistTracks && (
+          <PlayPauseTrackButton
+            size={40}
+            color={brandColors.pink.DEFAULT}
+            type={isThisTrackListPlaying ? "pause" : "play"}
+            onPress={handlePlayPausePress}
+          />
+        )}
+        <Text
+          style={{
+            fontSize: 24,
+            textAlign: "left",
+            flex: 1,
+          }}
+          numberOfLines={3}
+          bold
+        >
+          {playlistTitle}
+        </Text>
+        <Pressable onPress={handleMorePress}>
+          <MaterialCommunityIcons
+            name="dots-horizontal"
+            size={24}
+            color={colors.text}
+          />
+        </Pressable>
+      </View>
+      <FlatList
+        data={playlistTracks}
+        contentContainerStyle={{ flexGrow: 1 }}
+        renderItem={({ item, index }) => {
+          const { id, title, artist } = item;
+          const isLastRow = index === playlistTracks.length - 1;
+          const marginBottom = isLastRow ? height + 16 : 16;
+
+          return (
+            <TouchableOpacity onPress={() => handleRowPress(index)}>
+              <View
+                style={{
+                  flexDirection: "row",
+                  marginBottom,
+                  alignItems: "center",
+                  gap: 4,
+                }}
+              >
+                {/* TODO - swap placeholder with artwork of first track */}
+                <LogoIcon fill="white" width={60} height={60} />
+                {/* <SquareArtwork size={60} url={artworkUrl} /> */}
+                <View style={{ marginLeft: 10, flex: 1 }}>
+                  <Text
+                    style={{
+                      fontSize: 18,
+                    }}
+                    numberOfLines={3}
+                    bold
+                  >
+                    {title}
+                  </Text>
+                  <Text
+                    style={{
+                      fontSize: 18,
+                    }}
+                    numberOfLines={3}
+                    bold
+                  >
```

Comment:
To match styling on other pages:
```suggestion
                  <Text
                    numberOfLines={1}
                  >
```